weave/tests/trace/test_client_trace.py

```diff
@@ -2664,6 +2664,42 @@ def return_nested_object(nested_obj: NestedObject):
     assert call_result.output == nested_ref.uri()
 
 
+@pytest.mark.parametrize("batch_size", [1, 11, 101, 120])
```

Comment:
why these numbers?

Reply:
this tests batch of 10, 100, and 500. 120 because what if theres an off by one error and we are just getting lucky 🧠

Reply:
You should add a comment -- these look like magic numbers otherwise.  I would make the 10, 100, 500 consts and re-use them here with that comment

Reply:
Because of how the tests run synchronously generating 500 calls actually gets pretty expensive, like 60+ seconds expensive. 120 completed in 15 seconds ish. I will add a comment

Reply:
yeah you dont have to do 500, just comments RE: what these numbers mean